hawc/apps/assessment/forms.py

```diff
@@ -341,7 +341,8 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         obj = kwargs.pop("parent", None)
-        super().__init__(*args, **kwargs)
+        prefix = f"attachment-{kwargs.get("instance").pk if "instance" in kwargs else "new"}"
```

Comment:
Maybe add a random string or number to "new" as a quick fix here?

Reply:
This would require rewriting all the create tests to "get" the form, isolate the random number, and insert it into the data before POSTing. Since it's not an instant quick fix, I think tweaking our htmx viewsets next quarter is a better solution